plugins/bazaar/src/components/EntityBazaarInfoCard/EntityBazaarInfoCard.tsx

```diff
@@ -0,0 +1,345 @@
+            <MoreVertIcon />
+          </IconButton>
+        }
+        subheader={<HeaderIconLinkRow links={links} />}
+      />
+      <Divider />
+      <CardContent>
+        <Popover
+          open={popoverOpen}
+          onClose={popoverCloseHandler}
+          anchorEl={anchorEl}
+          anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+          transformOrigin={{ vertical: 'top', horizontal: 'right' }}
+        >
+          <MenuList>
+            <MenuItem
+              onClick={() => {
+                setOpen(true);
+                setPopoverOpen(false);
+              }}
+            >
+              <EditIcon className={classes.icon} />
+              <ListItemText primary="Suggest changes" />
+            </MenuItem>
+            <Divider />
+            <MenuItem
+              onClick={() => {
+                setOpenDelete(true);
+                setPopoverOpen(false);
+              }}
+            >
+              <DeleteIcon className={classes.icon} />
+              <ListItemText primary="Remove from Bazaar" />
+            </MenuItem>
+          </MenuList>
+        </Popover>
+        <Grid container>
+          <Grid item xs={12}>
+            <AboutField label="Announcement">
+              {bazaarProject.announcement
+                ? bazaarProject.announcement.split('\n').map((str: string) => (
+                    <Typography
+                      key={Math.floor(Math.random() * 1000)}
+                      variant="body2"
+                      paragraph
+                      className={classes.description}
+                    >
+                      {str}
+                    </Typography>
+                  ))
+                : 'No announcement'}
+            </AboutField>
+          </Grid>
+
+          <Grid item xs={6}>
+            <AboutField label="Status">
+              <StatusTag status={bazaarProject.status} />
+            </AboutField>
+          </Grid>
+
+          <Grid item xs={6}>
+            {' '}
+            <AboutField label="Latest members">
+              {members.length ? (
+                members.slice(0, 3).map(member => {
+                  return (
+                    <div key={member.userId}>
+                      <Avatar
+                        displayName={member.userId}
+                        customStyles={{
+                          width: '19px',
+                          height: '19px',
+                          fontSize: '8px',
+                          float: 'left',
+                          marginRight: '0.3rem',
+                        }}
+                      />
+                      <Link
+                        className={classes.memberLink}
+                        to={`http://github.com/${member.userId}`}
+                      >
+                        {member?.userId}
+                      </Link>
+                    </div>
+                  );
+                })
+              ) : (
+                <div />
```

Comment:
`null` is the canonical "render nothing" placeholder

Reply:
The reason we have an empty `div` is because with `null` it says "`Latest members: unknown`", and we would like it to be just an empty list. Is there another way to solve this? :)